src/cgame/rocket/rocketProgressBar.h

```diff
@@ -149,27 +149,14 @@ class RocketProgressBar : public Rocket::Core::Element
 	{
 		Element::OnPropertyChange( changed_properties );
 
-		if ( changed_properties.find( "color" ) != changed_properties.end() )
+		if ( changed_properties.Contains( Rocket::Core::PropertyId::Color ) )
 		{
-			color = Color::Adapt( GetProperty( "color" )->Get<Rocket::Core::Colourb>() );
+			color = Color::Adapt( GetProperty( Rocket::Core::PropertyId::Color )->Get<Rocket::Core::Colourb>() );
 		}
 
-		if ( changed_properties.find( "image" ) != changed_properties.end() )
```

Comment:
just checking that this is not accidental change and that I understand (could be useful some day): image is no longer a property, but an attribute?

Reply:
Possibly, I didn't reviewed the squashed PR but each commits one by one